### PR TITLE
feat: farm upgrades

### PR DESCRIPTION
**Pull request by GrowlingGrizzly**
Some things that need to be done:

* emojis
* balancing
* more ideas? maybe

### CODE REVIEW

File: src/utils/functions/economy/farm.ts [hunk 1]
```diff
@@ -117,11 +160,29 @@ export async function getClaimable(member: GuildMember | string, plantId: string
 
   for (const plant of plants) {
     const start = Date.now() - plant.harvestedAt.getTime();
-    const hours = start / 3600000; // hours - chatgpt
+    let hours = start / 3600000; // hours - chatgpt
+
+    const upgrades = getPlantUpgrades();
+    const userUpgrades = await getFarmUpgrades(id);
```
Comment (mxz7) [inline src/utils/functions/economy/farm.ts:166]: put it outside the loop so it doesn't fetch upgrades on every plant

File: package.json [hunk 2]
```diff
@@ -38,7 +38,8 @@
     "node-os-utils": "1.3.7",
     "p-all": "3.0.0",
     "sharp": "^0.33.5",
-    "string-similarity": "4.0.4"
+    "string-similarity": "4.0.4",
+    "twemoji-parser": "^14.0.0"
```
Comment (mxz7) [inline package.json:42]: this is bad package

**Review by mxz7 — CHANGES_REQUESTED** [review]
ok for balancing the sticks are gonna have to be suuuper rare.

and i think for the item thing it needs to be a lot. otherwise it's gonna be super easy to get

maybe silo should be a 10-15% upgrade too, since that will still be A LOT if you have a lot of plants. and probably about maybe 70 steel? not sure

**Comment by GrowlingGrizzly** [comment]
heres images for stick and hoes, idk what we want for a silo emoji

image
image
image
image
image
image

**Review by mxz7 — APPROVED** [review]

**Pull request merged**